service/project.go

```diff
@@ -707,7 +707,7 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	origProjectAliases, _ := model.FindAliasesForProject(id, "")
+	origProjectAliases, _ := model.FindAliasesForProject(id)
```

Comment:
Anything that's a method of UIServer is for the UI and so should use FindAlilasesForProjectFromDB; this includes this call and the following